roles/minecraft/defaults/main.yml

```diff
@@ -36,6 +36,7 @@ minecraft_web_domain: "{{ user.domain }}"
 
 minecraft_dns_record: "{{ lookup('vars', minecraft_name + '_web_subdomain', default=minecraft_web_subdomain) }}"
 minecraft_dns_zone: "{{ minecraft_web_domain }}"
+minecraft_dns_proxy: "false"
```

Comment:
This should be a true boolean without the quotes to match other places where we hardcode this.